tests/unit/src/commands/typedoc.ts

```diff
@@ -0,0 +1,128 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import loadModule, { cleanupModuleMocks } from '../../../_support/loadModule';
+import { spy, stub, SinonSpy, SinonStub } from 'sinon';
+
+let typedoc: any;
+let TypedocSpy: SinonSpy;
+let dirnameStub: SinonStub;
+let expandInputFilesStub: SinonStub;
+let extnameStub: SinonStub;
+let findConfigFileStub: SinonStub;
+let generateDocsStub: SinonStub;
+let generateJsonStub: SinonStub;
+let mkdirpStub: SinonStub;
+let statSyncStub: SinonStub;
+
+registerSuite({
+	name: 'commands/typedoc',
+
+	before() {
+		dirnameStub = stub();
+		expandInputFilesStub = stub();
+		extnameStub = stub();
+		findConfigFileStub = stub();
+		generateDocsStub = stub();
+		generateJsonStub = stub();
+		mkdirpStub = stub();
+		statSyncStub = stub();
+
+		const Typedoc = class {
+			bootstrapResult: any = { inputFiles: 'inputFiles' };
+			expandInputFiles: SinonStub = expandInputFilesStub;
+			generateDocs(project: any, out: any) {
```

Comment:
We should, as it would be more consistent with the rest of the class mocking, but please recall the conversation we had last week wherein I was attempting to do that and it was failing, due to the differences between having a property (alias of a function) assigned to a class vs a method on the prototype of the class. Your request cannot be achieved because of how the `typedoc` command uses the `generateDocs` and `generateJson` methods internally, requiring the methods to exist on the prototype.